Source/DafnyLanguageServer.Test/Performance/LargeFilesTest.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Dafny.LanguageServer.IntegrationTest.Extensions;
+using Microsoft.Dafny.LanguageServer.IntegrationTest.Util;
+using Microsoft.Dafny.LanguageServer.Workspace;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using Range = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Performance;
+
+public class LargeFilesTest : ClientBasedLanguageServerTest {
+  protected override Task SetUp(Action<DafnyOptions> modifyOptions) {
+    return base.SetUp(options => {
+      modifyOptions?.Invoke(options);
+      options.Set(ServerCommand.UpdateThrottling, ServerCommand.DefaultThrottleTime);
+    });
+  }
+
+  /// <summary>
+  /// To reduce runtime of this test,
```

Comment:
Can you explain why someone would like to reduce the runtime of this test? Or is it the runtime of the functions being tested that can be reduced?